libcontainer/label/label_selinux.go

```diff
@@ -28,9 +37,13 @@ func InitLabels(options []string) (string, string, error) {
 				return "", "", nil
 			}
 			if i := strings.Index(opt, ":"); i == -1 {
-				return "", "", fmt.Errorf("Bad SELinux Option")
+				return "", "", fmt.Errorf("Bad label option %q, valid options 'disable, user, role, level, type'", opt)
```

Comment:
Should we return a different message indicating that the expected format needs a colon?